SUMMARY.md

```diff
@@ -2,563 +2,563 @@
 
```

Comment:
This file is generated by gitbook, so I think any changes done here will be reverted by it

Reply:
I've modified the commit and removed this file, rebased, force pushed.